src/components/RecipientItem.vue

```diff
@@ -1,9 +1,10 @@
 <template>
   <div class="is-flex is-justify-content-flex-start" v-if="recipient">
     <div
-      class="pr-3 is-clickable is-align-items-center is-flex"
+      class="is-clickable is-align-items-center is-flex recipient-icon"
```

Comment:
removing the pr-3 is messing with the correct placement of the icon, is that expected ?